test/testcafe/spec/EnrollProfileView_spec.js

```diff
@@ -57,4 +64,15 @@ test.requestHooks(requestLogger, EnrollProfileSubmitMock)('should show submit bu
   requestLogger.clear();
   await t.expect(enrollProfilePage.getFormTitle()).eql('Sign in');
   await t.expect(await enrollProfilePage.getSaveButtonLabel()).eql('Submit');
+});
+
+test.requestHooks(requestLogger, EnrollProfileSignUpWithAdditionalFieldsMock)('should show more fields when creating a new user', async t => {
+  const enrollProfilePage = new EnrollProfileViewPageObject(t);
+  const identityPage = await setup(t);
+  await identityPage.clickSignUpLink();
+
+  requestLogger.clear();
+  await t.expect(await enrollProfilePage.getFormFieldLabel('userProfile.countryCode')).eql('Country code');
+  await t.expect(await enrollProfilePage.getFormFieldLabel('userProfile.timezone')).eql('Timezone');
```

Comment:
nit: Is it possible to verify that they're both of type 'select' or something to show that they're actually dropdowns as well? And maybe verifying that a value exists in it (e.g. `United States` for country, `EST` for timezone)

Reply:
new tests added for clicking and selecting dropdown values